src/apartment-preview/apartment-preview.component.html

```diff
@@ -0,0 +1,64 @@
+<!--<table mat-table [dataSource]="apartments" class="mat-elevation-z8 demo-table">
```

Comment:
There a lot of different comments and redundant comments, please remove them is they are really unused